Core/include/Acts/Utilities/BinUtility.hpp

```diff
@@ -328,4 +328,8 @@ class BinUtility {
   Transform3 m_itransform;                 /// unique inverse transform
 };
 
+void to_json(nlohmann::json& j, const BinUtility& bu);
```

Comment:
Minor suggestion, but would it make sense to have these parametrized on some kind of typedef? Something like `Acts::json` just in case we ever have to swap nlohmann out for another JSON library?